app/controllers/supervision/controller.rb

```diff
@@ -0,0 +1,3 @@
+class Supervision::Controller < ApplicationController
```

Comment:
Why need this controller? Is there any other controllers which will inherit this controller?